lib/releaselib.sh

```diff
@@ -450,6 +450,15 @@ release::gcs::copy_release_artifacts() {
   local gce_path=$release_stage/full/kubernetes/cluster/gce
   local gci_path
 
+  logecho "Publish release artifacts to gs://$bucket..."
+
+  logecho "- Checking whether $gcs_destination already exists..."
+  if $GSUTIL ls $gcs_destination >/dev/null 2>&1 ; then
```

Comment:
`gsutil ls` is only eventually consistent, but I'm going to assume this isn't really a big deal here.